src/views/Cards.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+  <v-container fluid grid-list-md>
+    <v-layout row wrap>
+      <v-flex v-show="lists.length === 0" xs12>
+        <v-card>
+          <v-card-title>Crie o seu primeiro card de tarefas!!!</v-card-title>
+        </v-card>
+      </v-flex>
+      <v-flex v-for="(card,index) in lists" :key="index" xs6>
+        <v-card>
+          <v-card-title primary-title>{{ card.title }}</v-card-title>
+          <v-card-text v-for="item in card.items" :key="item.id">
+            <v-checkbox v-if="item.type == 'checkbox'" :label="item.text"></v-checkbox>
```

Comment:
In this case you have a `v-checkbox` without content so we usually self close.

`<v-checkbox v-if="item.type == 'checkbox'" :label="item.text"/>`